dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatCompletionExtensions.cs

```diff
@@ -56,6 +59,7 @@ await foreach (var chatMessageStream in chatCompletionResult.GetStreamingChatMes
         {
             var chatMessage = await chatResult.GetChatMessageAsync(cancellationToken).ConfigureAwait(false);
             messageContent.Append(chatMessage.Content);
+            break;
```

Comment:
If this is always just one result, we should get rid of the foreach loop and the StringBuilder

Reply:
Agreed. Done.